lib/utils/filename-normaliser.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const path = require('path');
+
+const pkg = require(`${process.cwd()}/package.json`);
+
+module.exports = function normaliseFileName(fileName) {
```

Comment:
Please let me know if this seems like a good approach. Will add handling for projects with MODULE_UNIFICATION enabled if this looks like a good idea.